### ISSUE TITLE
Usage for haproxy latency metrics

### ISSUE THREAD

**Issue by matschaffer** [dylanmei/haproxy_mtail#1]
Thanks for starting this. Looks really useful in helping to understand our insight into haproxy.

Have you been able to use this to get something like a 99th percentile for requests going through haproxy?

**Comment by dylanmei** [comment]
No I'm not sure how I would do observations with mtail directly. I assume you're interested in this for something like Prometheus where it's lossy between scrapes.

**Comment by matschaffer** [comment]
Exactly. And I'm also not sure :) But prometheus clients have histogram buckets so seems like there must be some approach to it. Will let you know if I end up digging further.

**Comment by dylanmei** [comment]
That's great -- I'd appreciate knowing if you come up with something!
